great_tables/_formats.py

```diff
@@ -2250,6 +2250,255 @@ def fmt_datetime_context(
     return x_formatted
 
 
+def fmt_tf(
+    self: GTSelf,
+    columns: SelectExpr = None,
+    rows: int | list[int] | None = None,
+    tf_style: str = "true-false",
+    pattern: str = "{x}",
+    true_val: str | None = None,
+    false_val: str | None = None,
+    na_val: str | None = None,
+    colors: list[str] | None = None,
+) -> GTSelf:
+    """
+    Format True and False values
+
+    There can be times where boolean values are useful in a display table. You might want to express
+    a 'yes' or 'no', a 'true' or 'false', or, perhaps use pairings of complementary symbols that
+    make sense in a table. The `fmt_tf()` method has a set of `tf_style=` presets that can be used
+    to quickly map `True`/`False` values to strings, or, symbols like up/down or left/right arrows
+    and open/closed shapes.
+
+    While the presets are nice, you can provide your own mappings through the `true_val=` and
+    `false_val=` arguments. For extra customization, you can also apply color to the individual
+    `True`, `False`, and NA mappings. Just supply a list of colors (up to a length of 3) to the
+    `colors=` argument.
+
+    Parameters
+    ----------
+    columns
+        The columns to target. Can either be a single column name or a series of column names
+        provided in a list.
+    rows
+        In conjunction with `columns=`, we can specify which of their rows should undergo
+        formatting. The default is all rows, resulting in all rows in targeted columns being
+        formatted. Alternatively, we can supply a list of row indices.
+    tf_style
+        The `True`/`False` mapping style to use. By default this is the short name `"true-false"`
+        which corresponds to the words `"true"` and `"false"`. Two other `tf_style=` values produce
+        words: `"yes-no"` and `"up-down"`. The remaining options involve pairs of symbols (e.g.,
+        `"check-mark"` displays a check mark for `True` and an ✗ symbol for `False`).
+    pattern
+        A formatting pattern that allows for decoration of the formatted value. The formatted value
+        is represented by the `{x}` (which can be used multiple times, if needed) and all other
+        characters will be interpreted as string literals.
+    true_val
```

Comment:
In numpydoc style, I think you can do something like...

```
    true_val, false_val
        Some explanation for both
```

(But I could be wrong, and also maybe quartodoc is not handling it right 😅 )

Reply:
From pairing, it looks like this works (although the parameters still have separate lines in the parameter list). let's try to get this working. We might need to unpin griffe